class-tgm-plugin-activation.php

```diff
@@ -1258,7 +1274,8 @@ public function register( $plugin ) {
 		 *
 		 * @param string $source The source of the plugin as provided, either empty (= WP repo), a file path
 		 *                       (= bundled) or an external URL.
-		 * @return string        'repo', 'external', or 'bundled'
+		 *
+		 * @return string 'repo', 'external', or 'bundled'
```

Comment:
And again, we need to exclude another rule then - well at least in WPCS, I think we should keep it separate like this in TGMPA as we both prefer it ;-)